src/platform/datapath_epoll.c

```diff
@@ -2564,7 +2590,7 @@ CxPlatSocketSendInternal(
     while (SendData->SentMessagesCount < TotalMessagesCount) {
 
         int SuccessfullySentMessages =
-            sendmmsg(
+            CXPLAT_SENDMMSG(
```

Comment:
`cxplat_sendmmsg_shim` returns a `ssize_t` but `SuccessfullySentMessages` is an `int`.

Reply:
Ah, yeah I was looking at the wrong signature. Fixed cxplat_sendmmsg_shim